admin-dev/themes/new-theme/js/components/grid/extension/catalog/position-extension.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import tableDnD from "tablednd/dist/jquery.tablednd.min";
+
+const $ = window.$;
+
+/**
+ * Class CategoryPositionExtension extends Grid with reorderable category positions
+ */
+export default class CategoryPositionExtension {
```

Comment:
isnt it from #10160 pr? 🤔 did you make it reusable? name also contains `Category` name, same for docblock.

Reply:
yes I used your extension because I wasn't sure your PR would be merged, I generalized it and change the update call
It is a simpler version of your component since it does not use ajax calls
But you're right I forgot to rename it